pkg/sql/sem/tree/testdata/eval/int_min

```diff
@@ -2,4 +2,4 @@
 eval
 (-9223372036854775807:::int - 1) / -1
 ----
-9223372036854775808
+-9223372036854775808
```

Comment:
(not sure about this one. this is prohibited behaviour in postgres)

Reply:
Makes me think of #26932. Maybe worth filing a new issue about.